pkg/cli/cmd/app/connections/display_test.go

```diff
@@ -17,146 +17,119 @@ limitations under the License.
+					{
+						ID:        &sqlCntrID,
+						Direction: &dirInbound,
 					},
 				},
-				makeRedisResourceID("b"): {
-					node: nodeFromID(makeRedisResourceID("b")),
-					Connections: []connectionEntry{
-						{
-							Name: "B",
-							From: nodeFromID(containerResourceID),
-							To:   nodeFromID(makeRedisResourceID("b")),
-						},
-					},
-					Resources: []outputResourceEntry{
-						{
-							node:     nodeFromID(awsMemoryDBResourceID),
-							Provider: "aws",
-						},
+			},
+			{
+				ID:                &sqlCntrID,
+				Name:              &sqlCntrName,
+				Type:              &sqlCntrType,
+				ProvisioningState: &provisioningStateSuccess,
+				OutputResources:   []*corerpv20231001preview.ApplicationGraphOutputResource{},
+				Connections: []*corerpv20231001preview.ApplicationGraphConnection{
+					{
+						Direction: &dirOutbound,
+						ID:        &sqlRteID,
 					},
 				},
-				azureRedisCacheResourceID: {
-					node: nodeFromID(azureRedisCacheResourceID),
-					Connections: []connectionEntry{
-						{
-							Name: "C",
-							From: nodeFromID(containerResourceID),
-							To:   nodeFromID(azureRedisCacheResourceID),
-						},
+			},
+			{
+				ID:                &sqlDbID,
+				Name:              &sqlDbName,
+				Type:              &sqlDbType,
+				ProvisioningState: &provisioningStateSuccess,
+				OutputResources:   []*corerpv20231001preview.ApplicationGraphOutputResource{},
+			},
+			{
+				ID:                &sqlAppCntrID,
+				Name:              &sqlAppCntrName,
+				Type:              &sqlAppCntrType,
+				ProvisioningState: &provisioningStateSuccess,
+				OutputResources:   []*corerpv20231001preview.ApplicationGraphOutputResource{},
+				Connections: []*corerpv20231001preview.ApplicationGraphConnection{
+					{
+						Direction: &dirInbound,
+						ID:        &sqlDbID,
 					},
 				},
 			},
 		}
 
 		expected := `Displaying application: test-app
 
-Name: webapp (Applications.Core/containers)
-Connections:
-  webapp -> a (Applications.Datastores/redisCaches)
-  webapp -> b (Applications.Datastores/redisCaches)
-  webapp -> redis (Microsoft.Cache/Redis)
-  webapp -> error ('asdf-invalid-YO' is not a valid resource id)
-Resources:
-  demo (kubernetes: apps/Deployment)
-
-Name: a (Applications.Datastores/redisCaches)
+Name: sql-app-ctnr (Applications.Core/containers)
 Connections:
-  webapp (Applications.Core/containers) -> a
-Resources:
-  redis (azure: Microsoft.Cache/Redis) %s
+  sql-db (Applications.Datastores/sqlDatabases) -> sql-app-ctnr
+Resources: (none)
 
-Name: b (Applications.Datastores/redisCaches)
+Name: sql-ctnr (Applications.Core/containers)
 Connections:
-  webapp (Applications.Core/containers) -> b
-Resources:
-  redis-aqbjixghynqgg (aws: AWS.MemoryDB/Cluster)
+  sql-ctnr -> sql-rte (Applications.Core/httpRoutes)
+Resources: (none)
 
-Name: redis (Microsoft.Cache/Redis)
+Name: sql-rte (Applications.Core/httpRoutes)
 Connections:
-  webapp (Applications.Core/containers) -> redis
```

Comment:
What does the coverage look like after making this change?

I chose the test scenario for this to make sure that it covered all of the non-error codepaths 👍 

I don't have a big concern with changing the "test app" but I want to make sure it's covering all of the important cases.

You can see this easily by during a `run package tests` in vscode.

Reply:
I am getting the below result, suggesting this testcase might be OK. I just used this since I had deployed the bicep and recorded the values for struct earlier for the tests on core-rp API side.. not a strong reason :) please let me know if it's better to use the earlier test.

current branch: ok  	github.com/radius-project/radius/pkg/cli/cmd/app/connections	0.818s	coverage: 87.4% of statements

main branch: ok  	github.com/radius-project/radius/pkg/cli/cmd/app/connections	1.047s	coverage: 85.7% of statements

Reply:
That looks pretty good overall then. I'm not worried, I just wanted to understand the reason 👍